cmd/jira-lifecycle-plugin/server_test.go

```diff
@@ -3371,6 +3372,100 @@ In response to [this](https://github.com/org/repo/pull/1):
+			body:           "/verified remove",
+			verifiedRemove: true,
+			verificationInfo: []VerificationInfo{{
+				User:   "user",
+				Reason: "comment",
+				Type:   verifyRemoveType,
+				Org:    "org",
+				Repo:   "repo",
+				PRNum:  1,
+				Branch: "branch",
+			}},
+			options:        JiraBranchOptions{}, // no requirements --> always valid
+			labels:         []string{labels.JiraValidRef, labels.JiraValidBug, labels.SeverityCritical, labels.Verified},
+			expectedLabels: []string{labels.JiraValidRef, labels.JiraValidBug, labels.SeverityCritical},
+			expectedComment: `org/repo#1:@user: The ` + "`verified`" + ` label has been removed.
+
+<details>
+
+In response to [this](https://github.com/org/repo/pull/1):
+
+>/verified remove
+
+
+Instructions for interacting with me using PR comments are available [here](https://prow.ci.openshift.org/command-help?repo=org%2Frepo).  If you have questions or suggestions related to my behavior, please file an issue against the [openshift-eng/jira-lifecycle-plugin](https://github.com/openshift-eng/jira-lifecycle-plugin/issues/new) repository.
+</details>`,
+		},
+		{
+			name:           "verified remove comment results in verified later label being removed and bigquery data being uploaded",
+			issues:         []jira.Issue{{ID: "1", Key: "OCPBUGS-123", Fields: &jira.IssueFields{Project: jira.Project{Key: "OCPBUGS"}, Unknowns: tcontainer.MarshalMap{helpers.SeverityField: severityCritical}}}},
+			body:           "/verified remove",
+			verifiedRemove: true,
+			verificationInfo: []VerificationInfo{{
+				User:   "user",
+				Reason: "comment",
+				Type:   verifyRemoveLaterType,
+				Org:    "org",
+				Repo:   "repo",
+				PRNum:  1,
+				Branch: "branch",
+			}},
+			options:        JiraBranchOptions{}, // no requirements --> always valid
+			labels:         []string{labels.JiraValidRef, labels.JiraValidBug, labels.SeverityCritical, labels.VerifiedLater},
+			expectedLabels: []string{labels.JiraValidRef, labels.JiraValidBug, labels.SeverityCritical},
+			expectedComment: `org/repo#1:@user: The ` + "`verified-later`" + ` label has been removed.
+
+<details>
+
+In response to [this](https://github.com/org/repo/pull/1):
+
+>/verified remove
+
+
+Instructions for interacting with me using PR comments are available [here](https://prow.ci.openshift.org/command-help?repo=org%2Frepo).  If you have questions or suggestions related to my behavior, please file an issue against the [openshift-eng/jira-lifecycle-plugin](https://github.com/openshift-eng/jira-lifecycle-plugin/issues/new) repository.
+</details>`,
+		},
+		{
+			name:        "PR change results in verified label being removed and bigquery data being uploaded",
+			issues:      []jira.Issue{{ID: "1", Key: "OCPBUGS-123", Fields: &jira.IssueFields{Project: jira.Project{Key: "OCPBUGS"}, Unknowns: tcontainer.MarshalMap{helpers.SeverityField: severityCritical}}}},
+			body:        "/verified remove",
```

Comment:
Should `body` be set with a message on a "PR Change"?  Just want to make sure that the logic is processing the event and not the comment...

Reply:
Removed that from both tests